### ISSUE TITLE
Unable to remove templates

### ISSUE THREAD

**Issue by monkey-bars** [digitalpacman/foundryvtt-pf2e-reach-templates#2]
It says we can remove templates by continuing to hit R, but that just adds more on top of the existing templates. Possible to set alt+R or shift+R (or some modifier that isn't ctrl) to shrink it down to zero?

**Comment by digitalpacman** [comment]
Eventually it should disappear for you. Does it not do that? It goes 5-10-15-20-25-off.  Are you seeing something different? Let me know.  Maybe if you can send a video I can figure out what's wrong.

**Comment by gnlatgh** [comment]
I get the same issue. It will apply new templates, not removing the previous template and it does not automatically go away. So it get's spammy fast.

![image](https://user-images.githubusercontent.com/85176277/122356114-c5ef6300-cf52-11eb-97de-2cf23db4b798.png)
![image](https://user-images.githubusercontent.com/85176277/122356224-e0c1d780-cf52-11eb-9892-bc217a99771c.png)
![image](https://user-images.githubusercontent.com/85176277/122356492-25e60980-cf53-11eb-8f4b-08b9897a6465.png)

**Comment by digitalpacman** [comment]
I see. I'll add a check for this. But foundry was supposed to warn you,
that you don't have the dependent modules installed. You need to install
the attacher module. It's listed on the module page that it's required

**Comment by digitalpacman** [comment]
Hmm. Looks like you might have the dependent modules, but the wrong version maybe?  I only tested it with the newest.

**Comment by gnlatgh** [comment]
I am using token attacher 4.1.2 which came as a package with this mod (I installed and tested this mod like an hour ago so it's not an old setup) :)

**Comment by Cosmic-Lattee** [comment]
![Token Reach](https://user-images.githubusercontent.com/24766235/122449201-36a27980-cf6b-11eb-8972-d86b0a8fe48a.gif)
So this is a new game with only the modules needed for reach template to work. You might have to open the gif up in another page to see it all.

But I get the same problem as well.

**Comment by Cosmic-Lattee** [comment]
![Token Reach](https://user-images.githubusercontent.com/24766235/122450554-c0067b80-cf6c-11eb-9955-08ca5f6ac281.gif)

So I am having the same issue. This is a gif from a fresh game with only the required modules installed. You might have to open the gif up in another window to see all of it clearly.

But it seems to be the same issue that Monkey-bars and gnlatgh are suffering from.

**Comment by digitalpacman** [comment]
@Cosmic-Lattee do you know what your primary browser is? EDIT: Looks like chrome. Hmm.

**Comment by Cosmic-Lattee** [comment]
This is coming from the foundry game itself.

**Comment by digitalpacman** [comment]
@gnlatgh @Cosmic-Lattee @monkey-bars 

You can delete the unwanted boxes by running this in the console (f12) a bunch of times. You mighta gota refresh a few times.


canvas.drawings.placeables[0].delete()

**Comment by Cosmic-Lattee** [comment]
I mean you can delete them going to the Drawing Tools panel as well.

I tested this further by creating a second GM user in the game and tested in Chrome, Firefox, and Edge with the same result.

**Comment by digitalpacman** [comment]
I cannot fathom why it's failing for yall but works for me and others.

**Comment by JamzTheMan** [comment]
FWIW I get the exact same error, using Chrome and/or Windows client...

**Comment by digitalpacman** [comment]
I'm trying to update my foundry to the newest version to test this.  I was
running on a slightly older version. But the upgrade process is erroring.
Working on it.

**Comment by digitalpacman** [comment]
@monkey-bars @gnlatgh @Cosmic-Lattee @JamzTheMan 

I solved this issue. You'll need to upgrade to reach templates version 1.0.2

What was wrong:
FoundryVTT changed it's API for drawing items.  
Reach templates was supported in 0.7.x 
Reach templates was _not_ supported in 0.8.x (newest)

The upgrade of foundry broke the tool.

I changed reach templates to support 0.8.x and no longer support 0.7.x as I don't want to handle two versions of the code.
